src/core/market/params.ts

```diff
@@ -26,18 +26,18 @@ export namespace MarketParamChange {
 
 export interface MarketParamChanges {
   market?: {
-    poolrecoveryperiod?: number;
-    basepool?: Dec;
-    minstabilityspread?: Dec;
+    PoolRecoveryPeriod?: number;
+    BasePool?: Dec;
+    MinStabilitySpread?: Dec;
```

Comment:
I think these variable should start with lower case like `poolRecoveryPeriod`. Usually the upper case used for Classes.